src/assets/scripts/client/InputController.js

```diff
@@ -713,6 +714,16 @@ export default class InputController {
     }
 
     /**
+     * Disable default behavior of right-click menu appearing on right click
+     *
+     * @for InputController
+     * @method _disableRightClickMenu
+     */
+    _disableRightClickMenu() {
```

Comment:
For reference: 
[https://stackoverflow.com/questions/7477184/preventdefault-blocks-right-click-menu-in-firefox-on-mac-but-not-windows?rq=1](https://stackoverflow.com/questions/7477184/preventdefault-blocks-right-click-menu-in-firefox-on-mac-but-not-windows?rq=1)